pkg/accountGRPC/account.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package accountGRPC;
+option go_package = "github.com/stasBigunenko/monorepa/pkg/account/proto";
+
+import "google/protobuf/empty.proto";
+
+service UserGRPCService {
+  rpc Get(User_id) returns (Accounts) {}
+  rpc Create(User_id) returns (Obj) {}
+  rpc Update(Balance) returns (Balance) {}
+  rpc Delete(User_id) returns (google.protobuf.Empty) {}
+}
+
+message Obj {
+  string id = 1;
+  string user_id = 2;
+  string balance = 3;
+}
+
+message User_id {
```

Comment:
Use EITHER sankecase or camelcase, never mix. Here better use camelcase, in fact. Rename: UserID

Reply:
fixed